bundles/org.openhab.binding.phc/README.md

```diff
@@ -31,18 +19,55 @@ The binding was tested with QinHeng Electronics HL-340 USB-Serial adapter (RS485
 |                          | FTDI          | doesn´t work |
 |                          | on board      | bad          |
 | Up Board/ubilinux(Jessie)| HL-340        | not reliable |
-|                          | FTDI          | ok           |
+|                          | FTDI          | good         |
 
-Sometimes it is a bit slowly and you have to switch two times for reaction, but all in all it works ok.
-I'm trying to get a faster solution for communication in the next few weeks.
+If there are many modules on one bridge, the initialization can take a few minutes. If it not work you can plug in the modules one after the other.
+After initialization sometimes you have to switch two times or the reaction is a bit slowly but after you used a channel it works fine.
  
 For all devices running with Linux that use the ch341 driver (HL-340), the new version (ch34x) is needed.
 A guide how to install this can be found here: [CH340/341 UART Driver for Raspberry Pi](https://github.com/aperepel/raspberrypi-ch340-driver).  
 
 If you don´t have the same kernel as used in the guide you have to compile the module yourself. In the guide is described a specific way for the Raspberry Pi. With another Linux version you can go the normal way with linux-headers.   
 
+According to the [Wiki of the PHC-Forum](https://wiki.phc-forum.de/index.php/PHC-Protokoll_des_internen_Bus#USB_RS-485_Adapter) the newer version of the FTDI adapter doesn't really work anymore either.
+
 In Linux amongst others the user 'openhab' must be added to the group 'dialout': ```sudo usermod -a -G dialout openhab``` For more information read the [installation guide](https://www.openhab.org/docs/installation/linux.html#recommended-additional-setup-steps).
 
+### Connection
+
+There are two alternatives, the first of which is much simpler.
+
+#### Connection via power supply (simpler, preferred)
+
+The simplest way would be to connect the RS485 adaptor to the PHC power supply like in the table below and  Out at the power supply to the first module like the STM before.
+
+|  adaptor | PHC power supply |
+|----------|------------------|
+| 485+     | +A               |
+| 485-     | -B               |
+
+#### Make a direct RJ12 connection
+
+Connect a RJ12 plug with the RS485 adaptor and the power supply as follows.
+
+| RJ12 like in picture below | The cores on the other side |
+|----------------------------|-----------------------------|
+| 0V                         | 0V on power supply          |
+| B-                         | 485- on adaptor             |
+| A+                         | 485+ on adaptor             |
+| 24+                        | +24V on power supply        | 
+
+![RJ12 Connector](doc/RJ12-Connector.png)
+
+## Bridge
+
+The Bridge manages the communication between the things and the modules via a serial port (RS485). It represents the STM.
```

Comment:
Consider putting every sentence on a new line that is best practice for git-tracked mark-down files AFAIK.